flow/raster_cache.cc

```diff
@@ -112,14 +112,18 @@ bool RasterCache::UpdateCacheEntry(
 
 int RasterCache::MarkSeen(const RasterCacheKeyID& id,
```

Comment:
Super minor nit: Not sure about the out parameter. Perhaps a tuple or struct return. But your call.

Reply:
Make this return `std::tuple<int, bool>`.

Callers can then do

```cpp
int accesses;
bool has_image;
std::tie(accesses, has_image) = foo->MarkSeen(...);
```

and you don't have the out parameter anymore.

Reply:
I don't like returning unnamed parameters like that. Having the caller name them is bad practice - they API should describe its own return values. I used a struct instead.